packages/terra-props-table/src/PropsTable.jsx

```diff
@@ -32,18 +32,20 @@ function determineType(type) {
     }
   } else if (typeName === 'arrayOf') {
     if (type.value.name === 'shape') {
-      typeName = `array of objects structured like: ${formatShape(type.value.value)}`;
+      typeName = <span> array of objects structured like: <pre> {formatShape(type.value.value)} </pre></span>;
     } else {
       typeName = `array of ${type.value.name}s`;
     }
   } else if (typeName === 'union') {
     const options = type.value.map((option) => {
-      const name = option.name === 'shape' ? `an object structured like: ${formatShape(option.value)}` : option.name;
+      const name = option.name === 'shape' ? (<span key={option.value}> an object structured like:
+      <pre> {formatShape(option.value)} </pre></span>) :
+      (<span key={option.name}> {option.name}</span>);
```

Comment:
NIT but was stylelinter complaining about this? I feel line 42 & 43 should be indented.

Reply:
Linter wasn't complaining about it, I added indentation.